crates/astria-sequencer/Cargo.toml

```diff
@@ -32,3 +32,7 @@ tendermint-proto = { workspace = true }
 tendermint = { workspace = true }
 tokio = { workspace = true, features = ["rt", "tracing"] }
 tracing = { workspace = true }
+
+[features]
+default = ["faucet"]
+faucet = []
```

Comment:
At first I questioned whether a cfg feature is a good idea, but then I realized you don't want that in production.

I now realize that I would very much like some extra protection around this:

1. require in sequencer's `main` that a `DEVNET=1` env var be set (in other words, we must opt in not only with the feature flag, but also with an explicit config to activate the faucet).
2. in `main`, fire up a task that prints something like `THIS SEQUENCER HAS A FAUCET ACTIVE` every 30 seconds.